docs/integrations/ember-and-json-api.md

```diff
@@ -17,8 +17,30 @@ To solve this, in Ember, both the adapter and the serializer will need some modi
 
 ### Server-Side Changes
 
-there are multiple mimetypes for json that should all be parsed similarly, so
+First, set the adapter type in an initializer file:
+
+```ruby
+# config/initializers/ams_config.rb
+ActiveModelSerializers.config.adapter = :json_api
+```
+
+or:
+
+```ruby
+# config/initializers/ams_config.rb
+ActiveModelSerializers.config.adapter = ActiveModelSerializers::Adapter::JsonApi
+```
+
+You will also want to set the `key_transform` to `:underscore` since you will adjust the attributes in your Ember serializer to use underscores instead of dashes later.
```

Comment:
There actually isn't a reason to do this anymore, since ember accepts our default key transform without modification. It's still worth mentioning to make sure both server and client side apps have manching key transforms though.